src/components/Color.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Color({ name, red, blue, green, rgb }) {
```

Comment:
Your deconstructed import here won't work. You can deconstruct name and rgb, but not red, blue, green here. See Colors.js for why...